packages/rocketchat-drive/server/fetchFileFromDrive.js

```diff
@@ -0,0 +1,91 @@
+import {google} from 'googleapis';
+import Future from 'fibers/future';
+
+Meteor.methods({
+	async 'fetchFileFromDrive'(file) {
+		const future = new Future();
+		const driveScope = 'https://www.googleapis.com/auth/drive';
+
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid User', { method: 'uploadFileToDrive' });
```

Comment:
I noticed while browsing. Looks like method name is wrong in this file exceptions.

Reply:
It's been corrected, thanks @karakayasemi :+1: